web/theme.dart

```diff
@@ -0,0 +1,40 @@
+import 'dart:html';
+
+void init() {
+  var bodyElement = document.body;
+
+  if (bodyElement == null) {
+    return;
+  }
+
+  var theme = document.getElementById('theme') as InputElement;
+
+  theme.addEventListener('change', (event) {
+    if (theme.checked == true) {
+      bodyElement.setAttribute('class', 'dark-theme');
+      theme.setAttribute('value', 'dark-theme');
+      window.localStorage['colorTheme'] = 'true';
+    } else {
+      bodyElement.setAttribute('class', 'light-theme');
+      theme.setAttribute('value', 'light-theme');
+      window.localStorage['colorTheme'] = 'false';
+    }
+  });
+
+  void setChecked() {
```

Comment:
We should just inline this function body since it is only called once, immediately after it is declared.

Reply:
Addressed in the most recent commit